Kitodo-API/src/main/java/org/kitodo/api/dataformat/Workpiece.java

```diff
@@ -142,6 +148,15 @@ public void setStructure(Structure structure) {
         this.structure = structure;
     }
 
+    /**
+     * Returns the uplinks of this workpiece
+     * 
+     * @return the uplinks
+     */
+    public List<LinkedStructure> getUplinks() {
```

Comment:
how can there be multiple uplinks?

Reply:
Don't we need a "getDownlinks()"?

Reply:
The newspaper thing. First uplink: to the newspaper. Second uplink: to the year. The downlinks are realized by the `ExistingOrLinkedStructure` as linked structure, because they appear _inside_ the structure tree.